src/bilder/images/concourse/deploy.py

```diff
@@ -50,16 +51,23 @@
     VaultTemplate,
 )
 from bilder.components.hashicorp.vault.steps import vault_template_permissions
+from bilder.components.vector.models import VectorConfig
+from bilder.components.vector.steps import (
+    configure_vector,
+    install_vector,
+    vector_service,
+)
 from bilder.facts import has_systemd  # noqa: F401
 from bridge.lib.magic_numbers import (
+    CONCOURSE_DEFAULT_PROMETHEUS_PORT,
```

Comment:
Think the name is a little confusing here. Is it the default prometheus port or concourse or ...?

Reply:
So, it is the default port on which Concourse exposes Prometheus metrics according to the [table of port number allocations](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) in the Prometheus wiki. Happy to change the name if there's a way to make that clearer.

Reply:
How about `CONCOURSE_GRAPHITE_EXPORTER_PORT`?